generators/client/templates/gulpfile.js

```diff
@@ -126,6 +126,21 @@ gulp.task('inject', function () {
         .pipe(gulp.dest(config.app));
 });
 
+<%_ if(useSass) { _%>
+gulp.task('inject:sass', function () {
+    return gulp.src(config.scss + 'main.scss')
+        .pipe(inject(gulp.src(config.app + 'app/**/*.{scss,sass}').pipe(angularFilesort()), {
```

Comment:
Well maybe both config sass src and this path since if you would like to add component specific css you would most likely keep it along the other component files as John Papa  style guide suggests.